### PR TITLE
T-32182 Beethoven migration

### PR DESCRIPTION
**Pull request by couralex6**

### CODE REVIEW

**Comment by jeff-dude** [comment]
missing the `beethoven_x_optimism_trades.sql` file?

**Comment by couralex6** [comment]
> missing the `beethoven_x_optimism_trades.sql` file?

It's not in the lineage apparently: https://github.com/duneanalytics/spellbook/blob/a2065e75856c4271efcd2cdd4d5682d40c544f65/models/beethoven_x/beethoven_x_trades.sql#L10

We could still include it in this PR though

File: models/beethoven_x/optimism/beethoven_x_optimism_trades.sql [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{{ config(
+{{ config(tags=['dunesql'],
     schema = 'beethoven_x_optimism',
-    alias = alias('trades'),
-    partition_by = ['block_date'],
+    alias = 'trades',
```
Comment (jeff-dude) [inline models/beethoven_x/optimism/beethoven_x_optimism_trades.sql:3]: accidently removed alias macro here?

File: models/beethoven_x/optimism/beethoven_x_optimism_trades.sql [hunk 2]
```diff
@@ -1,11 +1,11 @@
-{{ config(
+{{ config(tags=['dunesql'],
```
[the remark this answers is not in the document]
Comment (couralex6) [reply models/beethoven_x/optimism/beethoven_x_optimism_trades.sql]: Good idea, just added it

File: models/beethoven_x/optimism/beethoven_x_optimism_trades.sql (hunk 1 above)
Comment (couralex6) [reply models/beethoven_x/optimism/beethoven_x_optimism_trades.sql]: oops, fixed!

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**